fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/DataConsumer.java

```diff
@@ -0,0 +1,20 @@
+package com.github.dedis.popstellar.utility.handler;
+
+import com.github.dedis.popstellar.model.network.method.message.data.Data;
+import com.github.dedis.popstellar.repository.LAORepository;
+import com.github.dedis.popstellar.utility.error.DataHandlingException;
+
+@FunctionalInterface
+public interface DataConsumer<T> {
```

Comment:
Maybe DataHandler would be a more intuitive name. What do you think ?

Reply:
If we use somthing that only has 2 arguments (context + data), we can use a BiConsumer instead of a custom interface for > 2 arguments.
And change DataConsumer<? extends Data> to BiConsumer<HandlerContext, ? extends Data>